src/raftstore/store/worker/apply.rs

```diff
@@ -1599,7 +1599,8 @@ impl ApplyDelegate {
 
     fn handle_put(&mut self, ctx: &ApplyContext, req: &Request) -> Result<Response> {
         let (key, value) = (req.get_put().get_key(), req.get_put().get_value());
-        check_data_key(key, &self.region)?;
+        // region key range has no data prefix, so we must use origin key to check.
```

Comment:
I think it is not necessary to add this comment everywhere, maybe just add it to the function documentation comment.

Reply:
Well, I think it's better to be verbose. We can remove them if you insist.